civis/parallel.py

```diff
@@ -392,6 +406,13 @@ def make_backend_template_factory(from_template_id,
                                   hidden=True):
     """Create a joblib backend factory that uses Civis Custom Scripts.
 
+    If your template has settable parameters "CIVIS_PARENT_JOB_ID" and
+    "CIVIS_PARENT_RUN_ID", then this executor will fill them with the contents
+    of the "CIVIS_JOB_ID" and "CIVIS_RUN_ID" of the environment which
+    created them. If the code doesn't have "CIVIS_JOB_ID" and "CIVIS_RUN_ID"
+    environment variables available, the child will not have
+    "CIVIS_PARENT_JOB_ID" and "CIVIS_PARENT_RUN_ID" environment variables.
```

Comment:
This seems like a lot of documentation to copy in multiple places.  Is there a single natural place to put this rather than copying across multiple functions?

Reply:
That's a fair criticism, but I don't know what single place would be best. Ideally, this will be a feature of Civis Platform more generally in the future, and users will expect it. For now, more places seems better. ... It occurs to me that users may not know that running scripts will always have a CIVIS_JOB_ID and CIVIS_RUN_ID. That /is/ a Platform feature. Should it be explained here?

Reply:
I think we want to avoid documenting all Platform features here (and rely on autogenerated docs where possible).  If you can't think of a better single place to put it, it's fine by me to keep as is (and sometimes repeating is better than the wrong abstraction!).